android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandler.kt

```diff
@@ -539,14 +539,20 @@ open class GestureHandler<ConcreteGestureHandlerT : GestureHandler<ConcreteGestu
   }
 
   open fun activate() {
-    if (state == STATE_UNDETERMINED || state == STATE_BEGAN) {
+    if (!manualActivation && (state == STATE_UNDETERMINED || state == STATE_BEGAN)) {
```

Comment:
I think I'd perhaps just add a parameter to `activate` that's called `force` and make it `false` by default. This should simplify things a bit, also allow to avoid calling `activate` from `activateManually` and also let overriding methods decide if they want to handle "force" parameter.

Reply:
Yeah it seems like a better idea, done in https://github.com/software-mansion/react-native-gesture-handler/pull/1717/commits/635ab9ddb01dcba83d669747c29fc48b6c5cdd1a.